cmd/zetae2e/local/utils.go

```diff
@@ -85,7 +85,7 @@ func waitKeygenHeight(
 	logger *runner.Logger,
 ) {
 	// wait for keygen to be completed
-	keygenHeight := int64(60)
+	keygenHeight := int64(55)
```

Comment:
Context? I used to have trouble by setting a value lower than 60. Don't mind leaving it like this if the test run.

Reply:
https://github.com/zeta-chain/node/pull/1791/commits/ce021e113520dbab64e1d27e5512c968b2dcefff